observability/logging/log_storage/about-log-storage.adoc

```diff
@@ -12,9 +12,7 @@ You can use an internal Loki or Elasticsearch log store on your cluster for stor
 [id="log-storage-overview-types"]
 == Log storage types
 
-Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as an alternative to Elasticsearch as a log store for the {logging}.
-
-Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly.
```

Comment:
I know you've not updated this section but because I mention moving "only" in a later comment, I wanted to mention it here. The placement of the word "only" can change the meaning of the sentence; for more information, please see the IBM Style Guide **Adverbs-only** section. (I'm not able to access the style guide online right now so I cannot provide a link.)

Reply:
Agreed. Thanks for suggesting this change.